src/layouts/base.njk

```diff
@@ -58,90 +58,93 @@
       </div>
     </section> 
         
-      <section id="projects" class="projects">
-        <h2>Projects</h2>
-        <div class="grid">
+      <section id="projects" class="cmp-projects">
+        <div class="cmp-projects__container">
+          <h2 class="cmp-projects__heading">Projects</h2>
 
-        <!-- neverland travelers -->
-        <div class="neverland-travelers">
-          <div class="left">
-            <div class="nt-text">
-              <h3>
-                Website for personal vlog
-              </h3>
-              <p>
-                I built and deployed the first version of a website dedicated to my personal vlog channel called Neverland Travelers. 
-              </p>
-              <div class="links">
-                <a href="work/neverland-travelers.html" class="link1">Learn More</a>
-                <a href="https://www.neverlandtravelers.com" target="_blank" class="link2">
-                  See Live Site &nbsp; &nbsp;  
-                  {% image 'src/public/usedresources/external-link.svg', '', 'Visit Neverland Travelers site' %}
-                </a>
+          <div class="obj-projects-grid">
+            {# neverland travelers vlog #}
+            <div class="obj-projects-grid__item obj-projects-grid__item--vlog">
+              {# text container on left #}
+              <div class="obj-projects-grid__item-text">
+                <h3 class="obj-projects-grid__heading">
+                  Website for personal vlog
+                </h3>
+                <p class="obj-projects-grid__description">
+                  I built and deployed the first version of a website dedicated to my personal vlog channel called Neverland Travelers. 
+                </p>
+                <div class="obj-projects-grid__links">
+                  <a href="/work/neverland-travelers.html" class="cmp-button__primary" aria-label="Learn more about the Neverland Travelers vlog project site">Learn More</a>
+                  <a href="https://www.neverlandtravelers.com" target="_blank" class="cmp-button__secondary">
+                    Visit Site
+                    {% image 'src/public/usedresources/external-link.svg', 'cmp-button__icon', 'Visit Neverland Travelers site' %}
+                  </a>
+                </div>
+              </div>
+
+              <div class="obj-projects-grid__item-screenshot">
+                {% image 'src/public/usedresources/NT-screenshot-hp.png', 'obj-projects-grid__image', 'Neverland Travelers Screenshot'%}
```

Comment:
> Neverland Travelers Screenshot

more helpful text could be like, 

> Neverland travelers homepage, with a header and several blog posts 

(I haven't looked at the screenshot yet so don't use that text word for word)

Reply:
Oooo yes, that is better. https://github.com/marissahuysentruyt/marissahuysentruyt/pull/26/commits/ba8afddf3bc9f9a507a351a3101acecf2c551538